### PR TITLE
chore: move adapters to new repo

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
## Changes

- https://github.com/withastro/adapters/pull/366
- https://github.com/withastro/adapters/pull/367

### CODE REVIEW

**Review by ematipico — DISMISSED** [review]
Blocking to prevent accidental merges. cc @matthewp this PR should be merged **without** Squash, so we can keep the commit history

**Comment by alexanderniebuhr** [comment]
I guess in theory this PR on the core repo, can be merged squashed **after** the other PRs are merged. But not 100% sure.

**Review by matthewp — APPROVED** [review]

**Pull request merged**
